source/uwp/AdaptiveCardTestApp/Expected/SourceCards/Tests.Bleed.Comprehensive.2087edd.json

```diff
@@ -0,0 +1,692 @@
+			"type": "Container",
+			"style": "emphasis",
+			"items": [
+				{
+					"type": "ColumnSet",
+					"style": "default",
+					"columns": [
+						{
+							"type": "Column",
+							"items": [
+								{
+									"type": "ColumnSet",
+									"columns": [
+										{
+											"type": "Column",
+											"items": [
+												{
+													"type": "Container",
+													"style": "emphasis",
+													"items": [
+														{
+															"type": "TextBlock",
+															"horizontalAlignment": "Center",
+															"text": "Y"
+														}
+													],
+													"bleed": true
+												}
+											],
+											"width": "stretch"
+										},
+										{
+											"type": "Column",
+											"items": [
+												{
+													"type": "Container",
+													"style": "emphasis",
+													"items": [
+														{
+															"type": "TextBlock",
+															"horizontalAlignment": "Center",
+															"text": "N",
+															"wrap": true
+														}
+													],
+													"bleed": true
+												}
+											],
+											"width": "stretch"
+										},
+										{
+											"type": "Column",
+											"items": [
+												{
+													"type": "Container",
+													"style": "emphasis",
+													"items": [
+														{
+															"type": "TextBlock",
+															"horizontalAlignment": "Center",
+															"text": "N"
+														}
+													]
+												}
+											],
+											"width": "stretch"
+										}
+									]
+								}
+							],
+							"width": "stretch"
+						},
+						{
+							"type": "Column",
+							"items": [
+								{
+									"type": "ColumnSet",
+									"columns": [
+										{
+											"type": "Column",
+											"items": [
+												{
+													"type": "Container",
+													"style": "emphasis",
+													"items": [
+														{
+															"type": "TextBlock",
+															"horizontalAlignment": "Center",
+															"text": "N"
+														}
+													],
+													"bleed": true
+												}
+											],
+											"width": "stretch"
+										},
+										{
+											"type": "Column",
+											"items": [
+												{
+													"type": "Container",
+													"style": "emphasis",
+													"items": [
+														{
+															"type": "TextBlock",
+															"horizontalAlignment": "Center",
+															"text": "N"
+														}
+													],
+													"bleed": true
+												}
+											],
+											"width": "stretch"
+										},
+										{
+											"type": "Column",
+											"items": [
+												{
+													"type": "Container",
+													"style": "emphasis",
+													"items": [
+														{
+															"type": "TextBlock",
+															"horizontalAlignment": "Center",
+															"text": "Y"
+														}
```

Comment:
Cool - am guessing this feeds into our unit tests?

Reply:
All of the test and sample files feed into our UWP test app which tests rendering, and i believe also into .NET tests which test parsing only.

---
In reply to: [279808458](https://github.com/Microsoft/AdaptiveCards/pull/2802#discussion_r279808458) [](ancestors = 279808458)